util/contentutil/fetcher.go

```diff
@@ -29,6 +30,10 @@ func (p *fetchedProvider) ReaderAt(ctx context.Context, desc ocispecs.Descriptor
 	return &readerAt{Reader: rc, Closer: rc, size: desc.Size}, nil
 }
 
+func (p *fetchedProvider) Info(ctx context.Context, dgst digest.Digest) (content.Info, error) {
```

Comment:
Did you check if this was needed for anything actually (apart from places in this PR that use it in cache import where it shouldn't be needed) or can `FromFetcher` keep returning `content.Provider`. This returns wrong size so not ideal if it actually gets called.

Reply:
I did check by replacing the `Info` implementation with a `panic` and running the buildkit tests. They all passed so I think it's not actually needed.

Also, I just implemented the `Info` for `DescriptorProviderPair` instead of doing it for each cache implementation.